editor.planx.uk/src/@planx/components/FileUploadAndLabel/model.ts

```diff
@@ -226,9 +236,9 @@ export const generatePayload = (fileList: FileList): Store.userData => {
 const getCachedSlotsFromPreviousData = (
   userFile: UserFile,
   previouslySubmittedData: Store.userData | undefined,
-): FileUploadSlot =>
+): FileUploadSlot[] =>
```

Comment:
This incorrect type was the root cause of this bug https://trello.com/c/aHHaTagP/2525-upload-and-label-throws-id-not-found-error-when-going-back-or-changing